INSTALLING.md

```diff
@@ -1,19 +1,62 @@
 # Installation and use
 
-Initially we use the command line and well-known systems (Java, R, Text-editing). 
+## Using Docker
 
-## AMI
+### Building your own image
+
+This repository comes with a [Dockerfile](https://github.com/petermr/openVirus/blob/master/Dockerfile) which builds the required environment (ami3, jdk8, node). This build and these instructions are heavily influenced by the Dockerfile for [ami3 itself](https://github.com/petermr/ami3/master/Dockerfile).
+
+To build the image yourself (assuming you have docker installed and running), you can use:
+
+```shell
+docker build --tag openvirus
```

Comment:
I tested this on Docker version 19.03.1, build 74b1e89e8a installed on Windows as per https://docs.docker.com/toolbox/toolbox_install_windows/#step-2-install-docker-toolbox Is it missing a `.` (period for cwd)? It worked when I added this to the command, building from the root of the this repo.